examples/app.py

```diff
@@ -2,6 +2,7 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2016-2018 CERN.
+# Copyright (C)      2022 TU Wien.
```

Comment:
The example app can be completely removed, we are removing them.

Reply:
Created an issue for this: https://github.com/inveniosoftware/invenio-stats/issues/123